src/vorta/views/repo_tab.py

```diff
@@ -88,6 +92,11 @@ def init_ssh(self):
         for key in keys:
             self.sshComboBox.addItem(f'{key["filename"]} ({key["format"]})', key['filename'])
 
+    def toggle_available_compression(self):
+        use_zstd = borg_compat.check('ZSTD')
+        self.repoCompression.model().item(1).setEnabled(use_zstd)
+        self.repoCompression.model().item(2).setEnabled(use_zstd)
```

Comment:
hardcoded 1 and 2 here is a potential future bug.

maybe determine offset somehow or add a comment above the place where the compression menu items are added to update this place in case of changes?

Reply:
I actually considered this, but finding the offset needs an extra statement. 

Since you noticed this as well, I added selecting the offset by value, rather than index.